src/test/suite/extension.test.ts

```diff
@@ -1,172 +1,21 @@
-import * as assert from 'assert';
-import path from 'path';
+import { assert } from 'chai';
 import nock from 'nock';
 
 // You can import and use all API from the 'vscode' module
 // as well as import your extension to test it
 import * as vscode from 'vscode';
-
 import * as myExtension from '../../extension';
-import { Uri } from 'vscode';
 
 suite('Extension Test Suite', () => {
-  // The vscode.Position class starts at 0.
+  // Note: The vscode.Position class starts at 0.
   // That is, the first line of a file is line 0, and
   // the first character of a line is character 0.
-  suite('#getDefinitionLocation', () => {
```

Comment:
I just want to make sure I understand this, but are these tests removed because they're now covered by checking the locations in the `FshDefinitionProvider` `scanAll` tests?

Reply:
Yes, that's the right idea. This function did a parse of all files in the workspace, and `scanAll` is now the function that does that. They run at different times due to the change in parsing strategy made to improve performance, so instead of directly returning a location, we expect the function to update the location maps.